docs/quickstart.rst

```diff
@@ -53,10 +53,22 @@ Running the Vagrant box
 
 The image is available in https://atlas.hashicorp.com/atomicapp/boxes/dev . However you dont have to download the box image manually. The following steps will take care of it implicitly.
 
+*This Vagrant box is built for VirtualBox. *
+
```

Comment:
@nitishb Actually the box is available for both libvirt and Virtualbox backend in https://atlas.hashicorp.com/atomicapp/boxes/dev. And these steps work fine in CentOS and Fedora with libvirt.  I think the issue comes when we try to run it in Ubuntu with libvirt backend. So the above statement is not valid